settings.py

```diff
@@ -2061,3 +2061,4 @@ def _pkce_required(client_id):
 GOOGLE_SHEETS_API_VERSION = "v4"
 
 DAYS_KEEP_GSHEET_STATUS = 14
+COMMCARE_ANALYTICS_HOST = ""
```

Comment:
This is specified for the staging and production environments respectively in [commcare-cloud](https://github.com/dimagi/commcare-cloud/pull/5474).

Reply:
nice